integrations/java/openhouse-java-runtime/src/main/java/com/linkedin/openhouse/javaclient/OpenHouseCatalog.java

```diff
@@ -88,6 +92,8 @@ public class OpenHouseCatalog extends BaseMetastoreCatalog
 
   protected Map<String, String> properties;
 
+  private Cache<TableOperations, FileIO> fileIOCloser;
```

Comment:
Just curious. How long the entries are present in the cache? Wondering if the cache will be populated with too many objects if there are more tables operation done in a short period of time. Do you think is going to consume more memory?